raiden/network/transport/matrix/client.py

```diff
@@ -227,6 +228,12 @@ def create_room(
     def get_presence(self, user_id: str) -> Dict[str, Any]:
         return self._send("GET", f"/presence/{quote(user_id)}/status")
 
+    def invite(self, room_id: RoomID, offer: Dict[str, str]) -> None:
+        self.send_message(room_id, json.dumps(offer), "m.call.invite")
```

Comment:
I thought we wouldn't use `m.call` any more?

Reply:
we do not use the specific `m.call.*` events anymore. 

`send_message` sends a message `m.room.message`. Each message contains a message type `msgType`. For normal text it is `m.text`. There exist also other supported types such as `m.notice`. But you can also set your own `msgType`. This is just an internal type for messages so clients have additional data to separate messages.

Reply:
Makes sense.